### ISSUE TITLE
FunctionDeclBitfields::UsesFPIntrin not serialized-- won't work in Serialized Modules?

### ISSUE THREAD

**Issue by erichkeane** [llvm/llvm-project#57429]
I noticed a few minutes ago that when @mibintc added the `UsesFPIntrin` flag to `FunctionDeclBitfields`, it doesn't seem to have made it to `ASTReaderDecl`/`ASTWriterDecl` serialization/deserialization.   This likely needs to be fixed.

**Comment by ChuanqiXu9** [comment]
If we added any new fields in the AST Nodes, we need to edit the ASTReader/ASTWriter too. The serialization can't make it automatically.

**Comment by erichkeane** [comment]
> If we added any new fields in the AST Nodes, we need to edit the ASTReader/ASTWriter too. The serialization can't make it automatically.

Yep! Found one we missed apparently though.  I didn't have time to do anything with it unfortunately, so captured it with the bug, hopefully someone can figure out how to write a good test for it.

**Comment by ChuanqiXu9** [comment]
Yeah, I can't make it immediately too. I filed an issue for it: https://github.com/llvm/llvm-project/issues/57439.
